spring-cloud-dataflow-server/docker-compose.yml

```diff
@@ -36,7 +40,7 @@ services:
         do
           sleep 1;
         done;
-        wget -qO- 'http://dataflow-server:9393/apps' --post-data='uri=http://bit.ly/Darwin-SR2-stream-applications-kafka-maven&force=true';
+        wget -qO- 'http://dataflow-server:9393/apps' --post-data='uri=http://bit.ly/Einstein-BUILD-SNAPSHOT-stream-applications-kafka-maven&force=true';
```

Comment:
https://github.com/spring-cloud/spring-cloud-dataflow/pull/2924/files#diff-386250650dabfd12fc198e7a788707bcR712 is using `Einstein-GA`

Reply:
+1